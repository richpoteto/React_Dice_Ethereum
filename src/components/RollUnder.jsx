import React from 'react';
import PropTypes from 'prop-types';

function RollUnder({ value }) {
  return (
    <div className="row">
      <div className="col">
        <h3>Roll under</h3>
      </div>
      <div className="col">
        <h3 className="roll-under-value">{value}</h3>
      </div>
    </div>
  );
}
RollUnder.propTypes = {
  value: PropTypes.number.isRequired,
};

export default RollUnder;
