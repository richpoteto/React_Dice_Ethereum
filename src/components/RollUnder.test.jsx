import React from 'react';
import RollUnder from './RollUnder';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<RollUnder value={51} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
