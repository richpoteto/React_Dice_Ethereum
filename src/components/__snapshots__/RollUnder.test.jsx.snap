// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="row"
>
  <div
    className="col"
  >
    <h3>
      Roll under
    </h3>
  </div>
  <div
    className="col"
  >
    <h3
      className="roll-under-value"
    >
      51
    </h3>
  </div>
</div>
`;
